helmclient.go

```diff
@@ -734,7 +735,7 @@ func getPod(client kubernetes.Interface, labelSelector, namespace string) (*core
 		return nil, microerror.Maskf(tooManyResultsError, "%d", len(pods.Items))
 	}
 	if len(pods.Items) == 0 {
-		return nil, microerror.Maskf(notFoundError, "%s", labelSelector)
+		return nil, microerror.Mask(notFoundError)
```

Comment:
This error is just matched and a tillerNotFoundError is returned which is matched in the operators.